contracts/TxRelay.sol

```diff
@@ -61,4 +69,44 @@ contract TxRelay {
     function getNonce(address add) public constant returns (uint) {
         return nonce[add];
     }
+
+    /*
+     * @dev Adds a number of addresses to a specific whitelist. Only
+     * the owner of a whitelist can add to it.
+     * @param listNum The list number
+     * @param allowedSenders the addresses to add to the whitelist
+     */
+    function addToWhitelist(uint listNum, address[] allowedSenders) public {
+        if (listOwner[listNum] == 0x0) {
+            listOwner[listNum] = msg.sender;
+        }
+        updateWhitelist(listNum, allowedSenders, true);
+    }
+
+    /*
+     * @dev Removes a number of addresses from a specific whitelist. Only
+     * the owner of a whitelist can remove from it.
+     * @param listNum The list number
+     * @param allowedSenders the addresses to add to the whitelist
+     */
+    function removeFromWhitelist(uint listNum, address[] allowedSenders) public {
+        updateWhitelist(listNum, allowedSenders, false);
+    }
+
+    /*
+     * @dev Internal logic to update a whitelist
+     * @param listNum The list number
+     * @param allowedSenders the addresses to add to the whitelist
```

Comment:
Given that this function can both add and remove senders, `allowedSenders` might not be the best name. Maybe change to `sendersToUpdate`